kod/object/passive/spell/persench/karahol.kod

```diff
@@ -96,6 +96,25 @@ messages:
       return iDuration;
    }
 
+   CanPayCosts(who = $, lTargets = $)
+   {
+      if lTargets = $
+      {
+         lTargets = [who];
+      }
+      
+      % check for enchantment already applied
+      if Send(first(lTargets),@IsEnchanted,#what=self)
+      {
+         Send(first(lTargets),@MsgSendUser,#message_rsc=KaraholsCurse_already_enchanted_rsc,
```

Comment:
Pass in `who` instead of `first(lTargets)` and this parm1 and parm2 are unnecessary -- only the following is needed:
```
Send(who,@MsgSendUser,#message_rsc=KaraholsCurse_already_enchanted_rsc);
```